pkg/apis/harvesterhci.io/v1beta1/lvm.go

```diff
@@ -0,0 +1,119 @@
+package v1beta1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ConditionType string
+type VGDesireState string
+type VGStatus string
+type VGType string
+
+const (
+	// VGStatusActive means the volume group is active
+	VGStatusActive VGStatus = "Active"
+	// VGStatusInactive means the volume group is inactive
+	VGStatusInactive VGStatus = "Inactive"
+	// VGStatusUnknown means the volume group is unknown
+	VGStatusUnknown VGStatus = "Unknown"
+
+	// VGStateEnabled means the volume group is enabled
+	VGStateEnabled VGDesireState = "Enabled"
+	// VGStateDisabled means the volume group is disabled
+	VGStateDisabled VGDesireState = "Disabled"
+	// VGStateReconciling means the volume group is reconciling
+	VGStateReconciling VGDesireState = "Reconciling"
+
+	// VGConditionReady means the volume group is ready
+	VGConditionReady ConditionType = "Ready"
+	// VGConditionAddDevice means the volume group is added a device
+	VGConditionAddDevice ConditionType = "AddDevice"
+
+	// ConditionTypeReady indicates the volume group is active
+	ConditionTypeActive ConditionType = "Active"
+	// ConditionTypeNotReady indicates the volume group is inactive
+	ConditionTypeInactive ConditionType = "Inactive"
```

Comment:
```suggestion
	// ConditionTypeActive indicates the volume group is active
	ConditionTypeActive ConditionType = "Active"
	// ConditionTypeInactive indicates the volume group is inactive
	ConditionTypeInactive ConditionType = "Inactive"
```

Just to sync the comment.

Reply:
Thanks, updated!